guides/source/getting_started.md

```diff
@@ -992,19 +992,19 @@ To understand how all of this works together, let's take another look at the
 `new` and `create` controller actions:
 
 ```ruby
-  def new
-    @article = Article.new
-  end
+def new
+  @article = Article.new
+end
 
-  def create
-    @article = Article.new(article_params)
+def create
+  @article = Article.new(article_params)
 
-    if @article.save
-      redirect_to @article
-    else
-      render :new, status: :unprocessable_entity
-    end
+  if @article.save
+    redirect_to @article
+  else
+    render :new, status: :unprocessable_entity
   end
+end
 ```
```

Comment:
I'm not sure about the other code blocks in this PR, but the indentation here was intentional.  The idea is that these lines look exactly how the appeared [in the code block further above](https://github.com/rails/rails/blob/9044d35c2f71e4e07cf28520adc2996a4284d2d2/guides/source/getting_started.md?plain=1#L896-L902), including indentation.